.eslintrc

```diff
@@ -9,6 +9,12 @@
     "unicorn/prefer-module": "off",
 
     // TODO: This can be removed if we decide to add jsdoc to all functions
-    "valid-jsdoc": "off"
+    "valid-jsdoc": "off",
+    "no-await-in-loop": "off",
+    "unicorn/filename-case": "off",
+    "no-negated-condition": "off",
```

Comment:
let's keep this and avoid using negated if checks. it's easy for bugs to creep in when this is used because the check isn't explicit. better to use explicit checks or we lose type safety